Doc/library/ctypes.rst

```diff
@@ -2007,6 +2033,20 @@ Utility functions
    function in the same library that allocated the memory.
 
 
+.. function:: dllist()
+   :module: ctypes.util
+
+   Try to provide a list of paths of the shared libraries loaded into the current
+   process.  These paths are not normalized or processed in any way. If the
+   function is not able to determine the list of loaded libraries (either because
+   the current platform does not support it, or because an error occurred), it
```

Comment:
If the current platform doesn't support it, we should at least document *why*. Could you add an excerpt about the system needing libc to have `dl_iterate_phdr` in order for this to work?

Reply:
I could, though I could imagine such a thing going stale if specific support for another platform was added down the line, or some other API was used

Reply:
Sure, and then we document that later on.